apps/workspaces/models.py

```
@@ -1,8 +1,11 @@
-from django.db import models
-from django_q.models import Schedule
-from django.db.models import JSONField
+from functools import cache
```

Pylint (4.0.0)

[error] Unrecognized option found: suggestion-mode, no-space-check

(E0015)